Src/Support/Google.Apis.Auth.AspNetCore3.IntegrationTests/Google.Apis.Auth.AspNetCore3.IntegrationTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Shouls this continue to be netcoreapp3.1?

Reply:
(Ignored as suggested.)